packages/backend/src/nest/libp2p/libp2p.service.ts

```diff
@@ -85,8 +131,8 @@ export class Libp2pService extends EventEmitter {
       libp2p = await createLibp2p({
         start: false,
         connectionManager: {
-          minConnections: 3, // TODO: increase?
-          maxConnections: 8, // TODO: increase?
+          minConnections: 1, // TODO: increase?
```

Comment:
Do you know what effect this has? Will it stop trying to auto-dial if it's connected to only one peer?

Reply:
Yea I updated it, I misunderstood the purpose of this parameter.